templates/overrides/navigation/links--language-block.html.twig

```diff
@@ -9,31 +9,26 @@
     - modal
 #}
 
-{% set _language = {
-  link: {
-    label: language.link.name|default(''),
-    href: language.link.href|default('#'),
-    target: language.link.target|default(language.modal.id|default('')),
-  },
-  modal: language.modal,
+{% set _language_link = {
+  label: language.link.name|default(''),
+  href: language.link.href|default('#'),
+  target: language.link.target|default(language.modal.id|default(''))
 } %}
-{% set extra_attributes = create_attribute() %}
-{% set extra_attributes = extra_attributes.setAttribute('data-bs-toggle', 'modal') %}
-{% set extra_attributes = extra_attributes.setAttribute('data-bs-target', '#' ~ _language.link.target) %}
-{% set extra_attributes = extra_attributes.addClass(
-  [
-    'text-center'
-  ]
-) %}
-  {% set title = _language.link.label %}
-  {% set link_title %}
-    <span class="d-block d-lg-inline-block">{{ title }}</span>
-  {% endset %}
+{% set extra_attributes = create_attribute({
+  'class': ['text-center'],
+  'data-bs-toggle': 'modal',
+  'data-bs-target': '#' ~ _language_link.target,
+  'aria-label': ('Change language. Current language is '|t) ~ _language_link.label ~ '.',
```

Comment:
this is not the way to use the t function, please add a placehoder and set _language_link.label as parameter

Reply:
yes, thanks